ldpd/init.c

```diff
@@ -229,7 +229,7 @@ send_capability(struct nbr *nbr, uint16_t capability, int enable)
 		 * Announcement Parameter in Capability messages sent to
 		 * its peers".
 		 */
-		/* FALLTHROUGH */
```

Comment:
I disagree that this should be a fallthrough to the below message.  The message seems wrong to me.